climpred/metrics.py

```diff
@@ -346,12 +510,14 @@ def _nrmse(forecast, reference, dim='svd', comparison=None):
     """
     rmse_skill = _rmse(forecast, reference, dim=dim)
     var = reference.std(dim)
+    if 'comparison' in kwargs:
```

Comment:
Same as above. Should be in the docstring and should have an else that throws an error since this is required.